core/src/main/kotlin/org/evomaster/core/search/service/mutator/StandardMutator.kt

```diff
@@ -258,16 +259,18 @@ open class StandardMutator<T> : Mutator<T>() where T : Individual {
 
 
             additionInfo!!.effectiveHistory.addAll(effective.mapNotNull {
-                if (it.individual.seeActions(isDb).isEmpty())
-                    ImpactUtils.findMutatedGene(it.individual.seeGenes(), gene, includeSameValue)
-                else
+                if (it.individual.seeActions(isDb).isEmpty()){
+                    if(it.individual is HttpWsIndividual) null
```

Comment:
i m not sure i understand the reasoning here... if something is `HttpWsIndividual`, then we do not collect history? but shouldn't the mutation of genes be independent from the type of individual? (as more types can be added in the future)